### PR TITLE
Ui profile 2.0

### PR DESCRIPTION
**Pull request by CheskaG**
Made ajustments, but I cannot properly adjust app for smaller windows.
preview1

Phone portrait view:
preview2

### CODE REVIEW

File: src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      )
+    }
+
+    function renderSectionBody(body: string) {
+      return (
+        <Typography align="right" variant="body1">
+          {body}
+        </Typography>
+      )
+    }
```
Comment (EdasL) [inline src/pages/Profile/Profile.tsx:53]: You shouldn't place functions inside of function

File: src/types/models/user.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export enum UserRole {
+  junior = 'junior',
+  mid = 'mid',
+  senior = 'senior',
+  teamLead = 'teamLead',
+  god = 'god',
+}
```
Comment (EdasL) [inline src/types/models/user.ts:7]: Enum params should start with a capital letter

**Comment by EdasL** [comment]
looks good boiiii, just a few comments and we are good to go!

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1 @@
+123,125
```
Comment (EdasL) [inline .vscode/settings.json:1]: Delete these, how they were added anyway 🤔  😄

File: src/types/models/user.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+export type User = {
+  name: string
+  lastname: string
```
Comment (EdasL) [inline src/types/models/user.ts:11]:
```suggestion
  lastName: string
```

File: src/types/models/user.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+export enum UserRole {
+  junior = 'junior',
+  mid = 'mid',
+  senior = 'senior',
+  teamLead = 'teamLead',
+  god = 'god',
```
Comment (EdasL) [inline src/types/models/user.ts:6]:
```suggestion
  Junior = 'junior',
  Mid = 'mid',
  Senior = 'senior',
  TeamLead = 'teamLead',
  God = 'god',
```

File: src/pages/Profile/Profile.tsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  Tab,
+  Typography,
+  Grid,
+  Paper,
+  Avatar,
+  Container,
+  Divider,
+  Hidden,
+  Tabs,
+} from '@material-ui/core'
```
Comment (EdasL) [inline src/pages/Profile/Profile.tsx:13]:
```suggestion
import React, { FC } from 'react'
import {
  Tab,
  Typography,
  Grid,
  Paper,
  Avatar,
  Container,
  Divider,
  Hidden,
  Tabs,
} from '@material-ui/core'
```

**Comment by CheskaG** [comment]
image

**Pull request merged**